docs/iot/iot-partners/index.md

```diff
@@ -0,0 +1,26 @@
+# Tizen IoT x Nubison Cloud
+
+Nubison cloud can readily be interconnected with Tizen IoT devices.
```

Comment:
Thanks for your comment, but your suggestion completely changes the original intention of the sentence.  

The intention was not to put an emphasize on the capability of seamless connectivity in Tizen Iot device, but rather, put an emphasize on the ability of Nubison Cloud to be interconnected with Tizen IoT devices with ease.

Reply:
How about going with: 'The Nubison cloud can readily be interconnected with Tizen IoT devices, which has an instrinsic capability of seamless connectivity.'?

Reply:
done

Reply:
@bson1012 the emphasis is still on interconnection, in my rewritten statement I am stressing on our product Tizen to have seamless connectivity options that why it can be readily connected to Nubison (interconnection)
Hope that clears my use of that statement .